pages/common/find.md

```diff
@@ -31,6 +27,10 @@
 
 `find {{root_path}} -name '{{*.ext}}' -exec {{wc -l {} }}\;`
 
-- Find files modified in the last 7 days, and delete them:
+- Find files modified in the last 7 days and delete them:
 
 `find {{root_path}} -mtime {{-7}} -delete`
```

Comment:
I think maybe the hyphen should be out of the syntax...
```suggestion
`find {{root_path}} -mtime -{{7}} -delete`
```

Reply:
Yes, it should be changed so good spot bl-ue